DOC Link field User manual (#234)
silverstripe/silverstripe-linkfield: docs/en/userguide/index.md

---

> title: Creating links in the CMS

Reviewer: ```suggestion
title: Managing links in the CMS
```

Author: Updated

> summary: How to use the Link Field module to create links in the CMS.

Reviewer: ```suggestion
summary: How to use the Link Field module to manage links in the CMS.
```

Author: Updated

---

> # Creating links in the CMS with LinkField module

Reviewer: ```suggestion
# Managing links in the CMS
```

Author: Updated

## Before we begin
> [!IMPORTANT]

> > Make sure that your SilverStripe CMS installation has the [LinkField](https://addons.silverstripe.org/add-ons/silverstripe/linkfield/) module installed.

Reviewer: addons doesn't exist anymore. Link to packagist.

Author: Updated

## Introduction

> In this section, we will delve into the process of creating links using the versatile LinkField module. This module offers the flexibility to generate links of diverse types, such as "Page on this site" link or "Link to a file". For further insights, refer to the ["Basic types of links"](01_User_manual/01_links_types.md) section.

Reviewer: ```suggestion
In this section, we will show the process of managing links using the LinkField module. This module provides a clean interface for creating and managing links in the CMS.
```
We're not trying to sell the module here, and we want people to be able to quickly find the information they need. I've reduced this to just a really basic explanation of what the module does, because by the time someone's reading this their digital agency has probably already installed the module for them and they're just looking for information about how to use the thing.

Author: Updated

![A screenshot of the link field on the page](_images/link_field.png)

> There are two primary types of Link field functionalities: one for crafting a single link and another for managing multiple links of similar or varying types. Each type of Link field will be comprehensively discussed in the subsequent section ["Essential functionalities associated with the Link field"](01_User_manual/02_work_with_link_field.md).

Reviewer: ```suggestion
There are two main types of link field: one for managing a single link and another for managing multiple links. Each type of Link field will be discussed in the ["Essential functionalities associated with the Link field"](01_User_manual/02_work_with_link_field.md) section.
```

Author: Updated

> The module streamlines the link creation process by providing a user-friendly form that includes all essential fields necessary for link generation. Additionally, it enables sorting capabilities and performs field validation based on the link type during saving.

Reviewer: ```suggestion
The module streamlines the link creation process by providing a user-friendly form that includes all essential fields necessary for link management.
```

Author: Updated
